src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1282,6 +1283,10 @@ bool entitiesQuery
       {
         exErr = invalidPolygon;
       }
+      else if (strncmp(exErr.c_str(), sortError, strlen(sortError)) == 0)
+      {
+        exErr = "Sort operation used more than the maximum RAM. You should create an index. Check Database Administration section at Orion documentation.";
```

Comment:
Fixed in b81c5b5f6972d57a6c67a2fe2828779b8344199f (doc) and 49b7e5f4d9f094144566e86fa89adc03b3c74e6b (code)